regress/pesterTests/sshconfig/ssh_config

```diff
@@ -0,0 +1,50 @@
+#	$OpenBSD: ssh_config,v 1.26 2010/01/11 01:39:46 dtucker Exp $
```

Comment:
Please call the folder testdata, consistent with how unittests are calling it. 

Also, put comments in this ssh_config file indicating its test usage (you could remove the rest of the comments in there)